File: DSA/Java/src/dsa/lib/utils/Nums.java
package dsa.lib.utils;

public class Nums {
    /**
     * Get the value on the place specified.
     *
     * @param num   The num.
     * @param place The place. 1 for unit place, 10 for tens place, 100 for hundreds place.
     * @return The value digit.
     */
    public static int placeValueOf(int num, int place) {
        return lastDigitOf(num / place);
    }

    public static int lastDigitOf(int num) {
        return num % 10;
    }
}

File: DSA/Python/src/dsa/lib/utils/nums.py
def lastDigitOf(num: int):
    return num % 10


def placeValueOf(num: int, place: int) -> int:
    """
    Get the value on the place specified.

    :param num: The num
    :param place: The place. 1 for unit place, 10 for tens place, 100 for hundreds place.
    :return: The value digit.
    """
    return lastDigitOf(num // place)
